presto-main/src/test/java/com/facebook/presto/operator/scalar/TestHyperLogLogFunctions.java

```diff
@@ -0,0 +1,148 @@
+package com.facebook.presto.operator.scalar;
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.google.common.base.Joiner;
+import com.google.common.collect.ImmutableList;
+import com.google.common.io.BaseEncoding;
+import io.airlift.slice.Slice;
+import io.airlift.stats.cardinality.HyperLogLog;
+import org.testng.annotations.Test;
+
+import java.util.Iterator;
+import java.util.List;
+
+import static com.facebook.presto.spi.type.BigintType.BIGINT;
+
+public class TestHyperLogLogFunctions
+        extends AbstractTestFunctions
+{
+    private TestHyperLogLogFunctions() {}
+
+    private static final int NUMBER_OF_BUCKETS = 32768;
+
+    @Test
+    public void testCardinalityNullArray()
+    {
+        assertFunction("cardinality(merge_hll(ARRAY[null, null, null]))", BIGINT, null);
+    }
+
+    @Test
+    public void testCardinalityMultipleNullColumns()
+    {
+        assertFunction("cardinality(merge_hll(ARRAY[null, null, null]))", BIGINT, null);
```

Comment:
What's the difference between this test and the one above?

Reply:
Should be 'assertFunction("cardinality(merge_hll(null))"'..., just fixed it.